tests/test_contracts.py

```diff
@@ -125,3 +125,16 @@ def test_scaffolder_extracts_events(scaffolder, test_filesystem):
     new_contract = scaffolder.from_abi(str(path), KNOWN_ADDRESS, "new_contract")
     new_contract.parse_events()
     assert new_contract.events
+
+
+def test_scaffolder_rejects_old_abi(scaffolder, test_filesystem):
```

Comment:
Is it only this abi it seems to have issues with? 
As i havent seen this issue before

Reply:
its only this one although i think that any old abi will look like it